src/olympia/abuse/tests/test_cinder.py

```diff
@@ -198,7 +193,7 @@ def test_build_report_payload(self):
             privacy_policy='Söme privacy policy',
             version_kw={'release_notes': 'Søme release notes'},
         )
-        message = '- bad addon!'
```

Comment:
What's the point of removing the characters in the test inputs? If they are allowed, shouldn't we keep them as examples of "working" features?